Experiment_Launcher_code/ExperimentLauncher.py

```diff
@@ -11,25 +11,26 @@ def main():
 
     if settings:
         # Extract the necessary parameters from the settings
+        experiment_name = settings.get('experiment_name')
+        comport_name = settings.get('comport_name')
         num_trials = settings.get('num_trials')
-        trial_duration = settings.get('trial_duration')
+        return_time = settings.get('return_time')
         decision_time = settings.get('decision_time')
-
-        opponent_type = settings.get('opponent_type')  # This should now be an OpponentType enum value
-        opponent_strategy_1 = settings.get('opponent_strategy')
-        opponent_strategy_2 = settings.get('computer_opponent_strategy')
-        # Adjust logic based on enum
-
-        num_trials += 2
```

Comment:
i need this or else it gets stuck in infinite loop in experiment manager

Reply:
Anushka, are you saying you need to add two trials to whatever the experimenter defines, or your software get stuck? This makes no sense. I do not see an issue when I test it. Maybe it was a problem in the old code that we fixed. Regardless, whatever the problem is, this cannot be the solution. if the experimenter asks for 2 trials you cannot run 4, just because you have a problem with 2 :-). try it again, and if you still see a problem we will fix it together on Thursday.

Reply:
sure we can have a look on thursday